ports/raspberrypi/bindings/cyw43/__init__.c

```diff
@@ -48,14 +47,42 @@ const mp_obj_type_t cyw43_pin_type = {
         )
 };
 
+//| PM_STANDARD: int
+//| """The standard power management mode"""
+//| PM_AGGRESSIVE: int
+//| """Aggressive power management mode for optimial power usage at the cost of performance"""
+//| PM_PERFORMANCE: int
+//| """Performance power management mode where more power is used to increase performance"""
+//| PM_DISABLED: int
+//| """Disable power management and always use highest power mode. CircuitPython sets this value at reset time, because it provides the best reliability."""
+//|
 //| def set_power_management(value: int) -> None:
 //|     """Set the power management register
 //|
-//|     According to Raspberry Pi documentation, the value 0xa11140
-//|     increases responsiveness at the cost of higher power usage.
+//|     For transmitter power, see ``wifi.Radio.txpower``.
+//|     This controls software power saving features inside the cyw43 chip.
+//|     it does not control transmitter power.
+//|
+//|     The value is interpreted as a 24-bit hexadecimal number of the form
+//|     ``0x00adbrrm``.
+//|
+//|     The low 4 bits, ``m``, are the power management mode:
+//|      * 0: disabled
+//|      * 1: aggressive power saving which reduces wifi throughput
+//|      * 2: Power saving with High througput
+//|
+//|     The next 8 bits, ``r``, specify "the maximum time to wait before going back to sleep" for power management mode 2. The units of ``r`` are 10ms.
+//|
+//|     The next 4 bits, ``b``, are the "wake period is measured in beacon periods".
+//|
+//|     The next 4 bits, ``d``, specify the "wake interval measured in DTIMs. If this is set to 0, the wake interval is measured in beacon periods".
+//|
+//|     The top 4 bits, ``a``, specifies the "wake interval sent to the access point"
 //|
-//|     Besides this value, there appears to be no other public documentation
-//|     of the values that can be used.
+//|     Several ``PM_`` constants gathered from various sources are included
+//|     in this module.  According to Raspberry Pi documentation, the value 0xa11140
+//|     (called `cyw43.PM_DISABLED` here) increases responsiveness at the cost of higher power
+//|     usage.
 //|     """
 STATIC mp_obj_t cyw43_set_power_management(const mp_obj_t value_in) {
     mp_int_t value = mp_obj_get_int(value_in);
```

Comment:
Does it make sense to pass multiple arguments for the pieces of this multi-value field, or a tuple? That could be a future enhancement; not needed now.

Reply:
I thought about it, but I don't think most users are going to care about this and they're better off not trying to mix and match. Honestly, I don't even really want to expose this, except while we're testing, if there's only one value that is reliable.

Reply:
We'll have other tools at our disposal as this port gets fleshed out:
• `wifi.radio.stop_station()` & `wifi.radio.stop_ap()`
• `wifi.radio.enabled = False`
(I finally bit the bullet and ordered a PPK2, hopefully can measure some of the differences)

Reply:
sounds fine